### ISSUE TITLE
[1.4.0beta3] Hard poweroff caused persistent corrupted display

### ISSUE THREAD

**Issue by ssssam** [polyend/TrackerBetaTesting#1183]
### Steps to reproduce

I accidentally removed power while using Tracker. (I can't remember what screen I was on ... possibly 'pattern', 'master' or 'sample recorder'). I was using firmware 1.4.0.beta3

On restart, the screen was corrupted - each vertical line of pixels was the same colour across the screen. Tracker otherwise worked normally, .e.g playing samples on the grid worked as normal. But the screen was an unusable set of vertical gray lines.

I tried:

  * Restarting device several times - didn't fix
  * Removing SDcard to a  PC and running fsck.fat - didn't fix (did find some SDcard errors)
  * Holding hidden 'reset' button and then booting without SD card - this did fix the issue.

### Other important information

* How often does this bug happen? (every time, often, "it only happened once")

Only once.

* Does this bug happen in previous firmware versions?

Restoring to 1.3.1 has fixed it.

* Does this bug happen in every project?

N/A

I hope this issue report is useful - reporting as I am on the beta version so potentially it's a new issue. However, I have accidentally removed power a lot of times and this is the first time I see any problem.

**Comment by bignutemporium1** [comment]
Hey ssssam,

Maybe try reverting to previous firmwares and seeing if this helps? Maybe start at base (1.0) and move up to 1.3.1

**Comment by ssssam** [comment]
Thanks, I have reverted to 1.3.1 which has resolved the issue. So it might be a bug in 1.4.0beta3

**Comment by ssssam** [comment]
I have experienced this problem again after PT crashed in firmware 1.3.1. Am trying a 2 minute power off to see if it resolves. If not, will try to reset firmware again.

**Comment by ssssam** [comment]
2 minute power off has resolved the issue again.

**Comment by apvanzanten** [comment]
I encountered this issue once a little while ago on 1.4.0. The unit froze up and didn't want to unfreeze or power off, so I unplugged the power, and then when I plugged it back in this display corruption occurred. I left it off and unpowered for a bit and then it worked fine again. I have since not had the issue. 

Is there anything known about what causes the issue? Is there some action I can/should take to make sure it doesn't happen again?

**Comment by MortalJS** [comment]
Hi @apvanzanten 

After some diffrent freezes  and crash. always try to of unit for 2 minutes. Try also diffrent power unit and cable. Unstable power from some bad cables can be the source of freezes.
For the moment I close the thread in order not to duplicate many that have already been created in this context.
